pkg/gvk/gvk_test.go

```diff
@@ -149,7 +149,7 @@ func Test_compareResources(t *testing.T) {
 							{
 								Name:       "deployments",
 								Namespaced: true,
-								Group:      "apps",
+								Group:      "extensions",
```

Comment:
@sseago why did this test need to change? Was it not doing what it claimed to do previously?

Reply:
@djwhatle So I first changed it when trying to debug an issue. But looking at it again (after realizing it wasn't really needed there), it looks like an error. `APIVersion` is "extensions/v1beta1". Group and Version of items in the `APIResources` slice should match this. I think it was a copy-and-paste oversight in the test data generation which doesn't really affect test results.